### PR TITLE
51 update building econ loss non structural content damage

### LINKED ISSUE #51: Update building econ loss to include non structural and content damage
Currently, buildingeconloss only supports structural loss. With additional mean damage inputs for acceleration sensitive, drift sensitive and content mean damages, total loss including structural and non-structural can be computed.

Also, update buliding_loss jupyter notebook example to demonstrate this analysis.

### PR DESCRIPTION
**Pull request by ncsa-mo**
close #51

### CODE REVIEW

**Review by navarroc — CHANGES_REQUESTED** [review]

File: pyincore/analyses/buildingeconloss/buildingeconloss.py [hunk 1]
```diff
@@ -93,6 +140,20 @@ def get_spec(self):
                     'description': 'result dataset name',
                     'type': str
                 },
+                {
+                    'id': 'component_type',
```
[the remark this answers is not in the document]
Comment (ncsa-mo) [reply pyincore/analyses/buildingeconloss/buildingeconloss.py]: Removed

File: pyincore/analyses/buildingeconloss/buildingeconloss.py [hunk 2]
```diff
@@ -76,6 +104,25 @@ def get_inflation_mult(self):
         """
         return (self.infl_factor / 100.0) + 1.0
 
+    def add_multiplies(self, dmg_set_df, occ_mult_df):
```
[the remark this answers is not in the document]
Comment (ncsa-mo) [reply pyincore/analyses/buildingeconloss/buildingeconloss.py]: Fixed typo

File: pyincore/analyses/buildingeconloss/buildingeconloss.py [hunk 3]
```diff
@@ -30,36 +30,64 @@ def __init__(self, incore_client):
...
+                loss = dmg_set_df["appr_bldg"].astype(float) * dmg_set_df["meandamage"].astype(float) * dmg_set_df["Multiplier"].astype(float) * infl_mult
+                lossdev = dmg_set_df["appr_bldg"].astype(float) * dmg_set_df["mdamagedev"].astype(float) * dmg_set_df["Multiplier"].astype(float) * infl_mult
 
-            bldg_results["strloss"] = valloss.round(2)
-            bldg_results["strlossdev"] = vallossdev.round(2)
+            bldg_results["loss"] = loss.round(2)
+            bldg_results["loss_dev"] = lossdev.round(2)
 
             result_name = self.get_parameter("result_name")
+            if comp_type == "str":
```
[the remark this answers is not in the document]
Comment (ncsa-mo) [reply pyincore/analyses/buildingeconloss/buildingeconloss.py]: Done

File: pyincore/analyses/buildingeconloss/buildingeconloss.py [hunk 4]
```diff
@@ -30,36 +30,49 @@ def __init__(self, incore_client):
 
     def run(self):
         """Executes building economic damage analysis."""
+
         # Get inflation input in %
         self.infl_factor = self.get_parameter("inflation_factor")
```
Comment (navarroc) [inline pyincore/analyses/buildingeconloss/buildingeconloss.py:35]: If you don't set an inflation value, I get the following error:

File "/home/cnavarro/git/pyincore/venv/lib/python3.8/site-packages/pyincore-1.2.0-py3.8.egg/pyincore/analyses/buildingeconloss/buildingeconloss.py", line 90, in get_inflation_mult
    return (self.infl_factor / 100.0) + 1.0
TypeError: unsupported operand type(s) for /: 'NoneType' and 'float'

Comment (ncsa-mo) [reply pyincore/analyses/buildingeconloss/buildingeconloss.py:35]: Done

**Review by navarroc — APPROVED** [review]

**Review by diegoac2 — APPROVED** [review]

**Pull request merged**